src/dcs/set_address_mode.rs

```diff
@@ -1,16 +1,59 @@
 //! Module for the MADCTL instruction constructors
 
+use super::DcsCommand;
+use crate::dcs::InterfaceExt;
 use crate::options::{
     ColorOrder, HorizontalRefreshOrder, MemoryMapping, ModelOptions, Orientation, RefreshOrder,
     VerticalRefreshOrder,
 };
 
-use super::DcsCommand;
+/// This is a public trait for user can change their MADCTL command.
+pub trait AddressMode: Copy {
```

Comment:
This trait adds a lot of additional trait bounds and I don't think we need it. See my comment for `Display::madctl` below for a possible alternative.